### PR TITLE
A couple of improvements to the README

### PR DESCRIPTION
**Pull request by benjaoming**
## Summary

Was wondering a bit about whether our README made clear: Public beta hasn't happened + NO DEPLOYMENTS are recommended.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -34,15 +40,28 @@ See https://learningequality.org/kolibri/ for more info.
...
 How can I contribute?
 ---------------------
 
 .. warning::
-  We welcome new contributors but since **Kolibri** is still in development, the API is not yet completely ready to integrate external plugins. Please start by:
+
+  *We welcome new contributors but since* ***Kolibri*** *is still in*
+  *development, the APIs are subject to change, and upgrade paths are not*
+  *guaranteed for all releases. We do not recommend deploying Kolibri before*
+  *the first public betas are available (for the very most urgent needs).*
```
Comment (radinamatic) [inline README.rst]: This `(for the very most urgent needs)` sounds a bit over-the-top... I'd recommend deleting it.

Comment (benjaoming) [reply README.rst]: But do we recommend deploying the first public beta? :) Okay, I can leave it out - of course things that we explicitly do not recommend, do not imply that we then recommend everything else :)

Comment (benjaoming) [reply README.rst]: Force-pushed a fix

**Comment by radinamatic** [comment]
> But do we recommend deploying the first public beta? :) Okay, I can leave it out - of course things that we explicitly do not recommend, do not imply that we then recommend everything else :)

I'm not against discouraging the deployment of betas, just saying that the expression `very most urgent needs` sounds over-the-top... :wink: 

Why don't we go with: `We strongly discourage deploying Kolibri before *the first public release* is made available.`?

**Comment by benjaoming** [comment]
@radinamatic knowing that many issues can show up that will prompt the software to be labelled "beta", yet still be suitable for deployment, I wouldn't "strongly discourage" anyone. I think the exact meaning of "beta" in the first releases will be evident as they happen.

**Comment by radinamatic** [comment]
Ok, now I'm totally confused as to what exactly you wanted to say, and what `very most urgent needs` actually refers too... :confused:

**Comment by benjaoming** [comment]
I was trying to phrase that in some cases, deploying the beta would be fine - deployment projects are always very timed - "now or never" -ish.. they won't wait for a more stable release and often can't upgrade later down the road.

**Comment by radinamatic** [comment]
I see... I think the phrase is good as it is now for that! :slightly_smiling_face:

**Pull request merged**